packages/vsce/__tests__/__e2e__/00_CreateNewGlobalTeamConfiguration.test.ts

```diff
@@ -0,0 +1,110 @@
+ * (the "License", destination); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ActivityBar, InputBox, DefaultTreeSection, ViewPanelAction, ModalDialog, WebElement, Workbench, EditorView } from "vscode-extension-tester";
+import { expect } from "chai";
+
+describe("Create New Global Team Configuration File Scenario", () => {
+  let cicsTree: DefaultTreeSection;
+  let quickPick: InputBox;
+  let dialog: ModalDialog;
+  let element: WebElement;
+  let plusIcon: ViewPanelAction;
+  let editorView: EditorView;
+
+  before(async function () {
+    this.timeout(15000);
+    // open the cics extension view
+    const view = await(await new ActivityBar().getViewControl("Zowe Explorer"))?.openView();
+
+    //handle notifications if config file is missing
+    const notifications = await new Workbench().getNotifications();
+    if (notifications.length > 0) {
+      await notifications[0].click();
+      const actions = await notifications[0].getActions();
+      await actions[0].click();
+
+      //select qp option
+      quickPick = await InputBox.create();
+      await quickPick.selectQuickPick(0);
+    }
+
+    cicsTree = await view.getContent().getSection("cics");
+    await cicsTree.click();
+  });
+
+  it("CICS Section title Check", async () => {
+    // title check for cics section
+    const title = await cicsTree.getTitle();
+    expect(title).equals("cics");
+  }).timeout(5000);
+
+  it("+ (plus) button clickable test", async () => {
+    // selecting cics view and clicking + icon
+    await cicsTree.click();
+    await cicsTree.expand();
+    plusIcon = await cicsTree.getAction(`Create a CICS Profile`);
+    expect(plusIcon).exist;
+    cicsTree.takeScreenshot();
+    await plusIcon.click();
+  }).timeout(5000);
+
+  it("Quick Pick Option test", async () => {
+    // Find quickpick
+    quickPick = await InputBox.create();
+    const qpItems = await quickPick.getQuickPicks();
+
+    const label1 = await qpItems[0].getLabel();
+    expect(label1).contains("＋ Create a New Team Configuration File");
+
+    if (qpItems.length > 1) {
+      const label2 = await qpItems[1].getLabel();
+      expect(label2).contains("Edit Team Configuration File");
+    }
+
+    const placeholder = await quickPick.getPlaceHolder();
+    expect(placeholder).equals('Choose "Create new..." to define or select a profile to add to the CICS tree');
+    cicsTree.takeScreenshot();
+  }).timeout(5000);
+
+  // now we can check what the dialog says
+  it("Create New Configuration File Test", async () => {
```

Comment:
This is more specific than the name suggests right? Doesn't it go through the motions of creating a new global config file, even though one already exists?